ONSdigital/design-system
# Fix/2631 cookie settings link override

### What is the context of this PR?
Fixes #2631 

### How to review
- Check the default URL for the cookie settings link can be overridden by macro option: `settingsLinkTextURL`
- Check updated test.
